cnxpublishing/views/admin.py

```diff
@@ -271,3 +534,150 @@ def admin_edit_site_message_POST(request):
     args = admin_edit_site_message(request)
     args['response'] = "Message successfully Updated"
     return args
+
```

Comment:
No, I'm not referring to the code just above this, but this actual green change block. I posted this comment at the top of it. Sorry for the confusion.

You'll see that the functions that follow are the functions that you already defined above.